[CI] JvmStatsTests testJvmStats failing

**Build Scans:**
- [elasticsearch-periodic #4725 / openjdk21_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/biu3xcdl4hpcu)
- [elasticsearch-periodic-platform-support #4650 / oraclelinux-7_platform-support-unix](https://gradle-enterprise.elastic.co/s/isvygxhms37rg)

**Reproduction Line:**
```
./gradlew ":server:test" --tests "org.elasticsearch.monitor.jvm.JvmStatsTests.testJvmStats" -Dtests.seed=64E293D9554CBE51 -Dtests.locale=en-ZW -Dtests.timezone=Pacific/Tongatapu -Druntime.java=21
```

**Applicable branches:**
8.16

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.monitor.jvm.JvmStatsTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testJvmStats),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: a value greater than <0L>
     but: <0L> was equal to <0L>
```

**Issue Reasons:**
- [8.16] 2 failures in test testJvmStats (0.7% fail rate in 303 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -196,9 +196,6 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=ml/inference_crud/Test delete given model referenced by pipeline}
   issue: https://github.com/elastic/elasticsearch/issues/115970
-- class: org.elasticsearch.monitor.jvm.JvmStatsTests
-  method: testJvmStats
-  issue: https://github.com/elastic/elasticsearch/issues/115711
 - class: org.elasticsearch.repositories.s3.S3ServiceTests
   method: testRetryOn403RetryPolicy
   issue: https://github.com/elastic/elasticsearch/issues/115986

--- a/server/src/test/java/org/elasticsearch/monitor/jvm/JvmStatsTests.java
+++ b/server/src/test/java/org/elasticsearch/monitor/jvm/JvmStatsTests.java
@@ -53,7 +53,7 @@ public class JvmStatsTests extends ESTestCase {
         assertThat(memoryPools, hasKey("Metaspace"));
         assertThat(memoryPools.keySet(), hasSize(greaterThan(3)));
         for (JvmStats.MemoryPool memoryPool : memoryPools.values()) {
-            assertThat(memoryPool.getUsed().getBytes(), greaterThan(0L));
+            assertThat("Memory pool: " + memoryPool.getName(), memoryPool.getUsed().getBytes(), greaterThanOrEqualTo(0L));
         }
 
         // Threads

